set.mm

```diff
@@ -30509,6 +30509,15 @@ generally appear in a single form (either definitional, but more often
        NM, 16-Nov-1995.)  (Proof shortened by Steven Nguyen, 5-May-2023.) $)
     raleqbi1dv $p |- ( A = B -> ( A. x e. A ph <-> A. x e. B ps ) ) $=
       ( wceq id raleqbidv ) DEGZABCDEJHFI $.
+    $( $j usage 'raleqbi1dv' avoids 'ax-12'; $)
+
+    $( Proof of ~ raleqbi1dv depending on ~ ax-12 , but not on ax-8.  Some
+       theorems, like ~ ralrab2 , profit from this swap.  (Contributed by Wolf
+       Lammen, 22-Sep-2024.)  (Proof modification is discouraged.)
+       (New usage is discouraged.) $)
+    raleqbi1dvALT $p |- ( A = B -> ( A. x e. A ph <-> A. x e. B ps ) ) $=
+      ( wceq cv wcel wi wal wral eleq2w2 imbi12d albidv df-ral 3bitr4g ) DEGZCH
+      ZDIZAJZCKSEIZBJZCKACDLBCELRUAUCCRTUBABCDEMFNOACDPBCEPQ $.
```

Comment:
`$( $j usage 'raleqbi1dvALT' avoids 'ax-8'; $)`should be added here, shouldn't it?

Reply:
I don't know.  Since a proof modification is not (easily) possible, I did not add it.